### PR TITLE
IQ-GPT in navbar

### PR DESCRIPTION
**Pull request by Aliiiu**
# IQ-GPT in navbar

This PR works on including iqgpt in the navbar and wiki page

## Linked issues

closes https://github.com/EveripediaNetwork/issues/issues/1759

### CODE REVIEW

**Comment by Softdev1** [comment]
Please reverse the arrangement to the old arrangement @Aliiiu    Hero section ==> Rank Page==> Iqgpt, Once the new arrangement is approved, I will let you know

**Comment by Aliiiu** [comment]
> Please reverse the arrangement to the old arrangement @Aliiiu Hero section ==> Rank Page==> Iqgpt, Once the new arrangement is approved, I will let you know

Alright, done

File: src/components/Wiki/WikiPage/WikiAdCard.tsx [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import React from 'react'
+
+const WikiAdCard = () => {
+  return (
+    <Box
+      rounded={'lg'}
+      w="full"
+      border={'1px'}
+      borderColor={'iqgptAdCardBorder'}
+      p={'12px'}
```
Comment (Adebesin-Cell) [inline src/components/Wiki/WikiPage/WikiAdCard.tsx:13]: ``p={3}``

File: src/components/Wiki/WikiPage/WikiInsights.tsx [hunk 2]
```diff
@@ -77,63 +78,74 @@ const WikiInsights = ({ wiki, ipfs, dateTime }: WikiInsightsProps) => {
       borderLeftWidth={{ base: 0, xl: '1px' }}
       p={{ base: 0, md: 2, xl: 4 }}
       pr={{ md: 11, xl: 4 }}
-      pt={{ xl: '24', md: '8', base: '10' }}
+      pt={{ xl: '36px', md: '8', base: '10' }}
```
Comment (Adebesin-Cell) [inline src/components/Wiki/WikiPage/WikiInsights.tsx:81]: ``xl=9``

**Review by Adebesin-Cell — APPROVED** [review]

**Review by Simplypheyie96 — APPROVED** [review]

**Pull request merged**